scripts/test_translation_templates.py

```diff
@@ -0,0 +1,127 @@
+import coloredlogs
+
+import os
+import docker
+import sys
+import subprocess
+import tempfile
+import logging
+
+logger = logging.getLogger(__name__)
+
+coloredlogs.install(level='DEBUG', logger=logger, fmt='%(message)s')
+
+SCHEMAS_TO_EXTRACT = ["census_individual_gb_wls", "census_individual_gb_nir"]
+
+
+def get_template_content(filename, ignore_context=False):
+    line_beginnings_to_ignore = ['"POT-Creation-Date']
+
+    if ignore_context:
+        line_beginnings_to_ignore += ["#:"]
+
+    with open(filename) as file:
+        return "".join(
+            filter(
+                lambda l: all(
+                    not l.startswith(param) for param in line_beginnings_to_ignore
+                ),
+                file.readlines(),
+            )
+        )
+
+
+def print_filename_results(filename, success=True):
+    if success:
+        logger.debug(f"{filename} - NO CHANGES")
+    else:
+        logging.error(f"{filename} - CHANGES FOUND")
+
+
+def build_static_template(output_filepath):
+    subprocess.run(
+        [
+            'pipenv',
+            'run',
+            'pybabel',
+            'extract',
+            '-F',
+            'babel.cfg',
+            '-o',
+            output_filepath,
+            '.',
+        ]
+    )
+
+
+def check_static_template(source_dir, target_dir):
+    source_contents = get_template_content(
+        f"{source_dir}/messages.pot", ignore_context=True
+    )
+    target_contents = get_template_content(
+        f"{target_dir}/messages.pot", ignore_context=True
+    )
+
+    success = source_contents == target_contents
+
+    print_filename_results(f"{source_dir}/messages.pot", success)
+
+    return 0 if success else 1
```

Comment:
Just return the boolean?
```suggestion
    return success
```